### PR TITLE
Add support for flipped FlxFrames

### PR DESCRIPTION
**Pull request by larsiusprime**
Adding support for flipped FlxFrames. This lets you get your atlases packed even tighter by re-using mirror images.

### CODE REVIEW

**Comment by sruloart** [comment]
Good thinking! talking about tighter packing, do you think your [MaxRects](https://github.com/larsiusprime/tdrpg-tools/blob/34c32fda9908d904f747d215a8e66d8760b5f9eb/dqsprite/src/MaxRectsBinPack.hx) implementation can work with FlxAtlas? currently it's missing a proper packing algorithm.

**Comment by larsiusprime** [comment]
Yeah!

I just added a readme:
https://github.com/larsiusprime/tdrpg-tools/tree/master/dqsprite

The tool itself is a bit hacky and designed specifically for defender's quest, but you can probably just look at what it's doing and get most of what you need right from there. I set it up to export a sparrow-compatible xml file and then feed it straight into FlxAtlas. It's working in my game right now.

**Comment by Gama11** [comment]
I'll let @Beeblerox review this one.

**Comment by larsiusprime** [comment]
@sruloart:

My EntitySprite class in flixel-editors (more like flixel-lars'-weird-experiments, lol) makes use of this if you want more examples

https://github.com/HaxeFlixel/flixel-editors/blob/dev/flixel/editors/EntitySprite.hx

File: flixel/graphics/frames/FlxAtlasFrames.hx [hunk 1]
```diff
@@ -256,6 +258,8 @@ class FlxAtlasFrames extends FlxFramesCollection
 			name = texture.att.name;
 			trimmed = texture.has.frameX;
 			rotated = (texture.has.rotated && texture.att.rotated == "true") ? true : false;
+			flipX = (texture.has.flipX && texture.att.flipX == "true") ? true : false;
```
Comment (Gama11) [inline flixel/graphics/frames/FlxAtlasFrames.hx]: No need for the ternaries here, these are already boolean expressions. :)

**Pull request merged**